games-simulation/openttd/openttd-1.10.0.ebuild

```diff
@@ -0,0 +1,180 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit xdg
+
+MY_PV="${PV/_rc/-RC}"
+MY_P="${PN}-${MY_PV}"
+
+DESCRIPTION="OpenTTD is a clone of Transport Tycoon Deluxe"
+HOMEPAGE="http://www.openttd.org/"
```

Comment:
https://www.openttd.org/

Reply:
Thank you for review, what should I do next?  (my first ebuild sumbitted to Gentoo via PR)